packages/buildessential.rb

```diff
@@ -119,6 +119,10 @@ class Buildessential < Package
   # Samurai
   #depends_on 'samurai'
 
+  # Tar
+  # Version > 1.30 needed for zstd support
+  depends_on 'tar'
```

Comment:
I'm not exactly a fan of needing an updated tar to compress files.  It would be better to use all of the out-of-the-box versions instead but...

Reply:
Only `i686` needs an updated tar. Other systems appear to have a new enough tar. Let me change that...

Reply:
The updated tar is needed for `decompressing` though...

Reply:
Changed the dependency to i686 only